config/notification-service/database.properties

```diff
@@ -0,0 +1,35 @@
+# Database connection settings.
+# To change the database provider, change the driver (hibernate.connection.driver_class),
+# the URL (hibernate.connection.url), and don't forget the dialect (hibernate.dialect)
+
+hibernate.connection.driver_class=org.hsqldb.jdbc.JDBCDriver
+hibernate.connection.url=jdbc:hsqldb:hsql://localhost:9001/notification-service
```

Comment:
I don't like much having `-` in a DB name. 
Why not call it `notification`
@AlexFalessi @fviale

Reply:
I dont mind